### PR TITLE
Avoid running `npm run build` when executing `npm install` locally

### PR DESCRIPTION
**Pull request by marionebl**
Employs [in-publish](https://www.npmjs.com/package/in-publish) to avoid running `npm run build` after each and every local `npm install`

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "commit": "node bin/git-cz",
     "build": "babel src --out-dir dist",
     "build:watch": "babel --watch src --out-dir dist",
-    "prepublish": "npm run build",
+    "prepublish": "not-in-install && npm run build || echo ''",
```
Comment (LinusU) [inline package.json]: Maybe replace `echo ''` with just `true`?

**Comment by LinusU** [comment]
@jimthedev sounds reasonable to me, any thoughts?

**Comment by jimthedev** [comment]
Should be ok, does it make sense to do a major version for this? I can't
think of a reason it'd be a breaking change.

**Comment by jimthedev** [comment]
@LinusU On second thought I think we should be good to release this as a bug fix as the original commit implies.

**Comment by marionebl** [comment]
This correctly should not trigger a semantic release run, as there is no possibility this affects `--production` library consumers at all, right?

Any reasons to hold back a merge into master?

File: package.json (hunk 1 above)
Comment (marionebl) [inline package.json]: Implemented your suggestion

**Comment by LinusU** [comment]
Thanks, as said this shouldn't affect the version number at all since it only affects local development...

**Pull request merged**